sections/csgcn/csgcn-chapter.tex

```diff
@@ -329,12 +329,14 @@ \section{CSGCN-ADJ}\label{sec:csgcn_adj}
 This section describes the second proposed model, CSGCN with Context in Adjacency Matrix (CSGCN-ADJ).
 
 \subsection{Model intuition}\label{subsec:csgcn_adj_intuition}
-With the second proposed model, context is included in the convolution by adding context nodes to the graph, which will be used to represent interactions between users and items in a given context as a tuple $(u,i,C)$, implying that a user $u$ has interacted with item $i$ in context $C$.
+With the second proposed model, context is included in the convolution by adding context nodes to the graph, which will be used to represent interactions between users and items in a given context as a tuple $(u,i,C')$, implying that a user $u$ has interacted with item $i$ in context $C'$.
 The adjacency matrix will now include interactions, what context the users have interacted in, and the context in which items have been interacted with.
-This differs from \textit{CSGCN-IS} since we now model context in a similar fashion to side-information, where each context feature has a single vector representation, instead of having a vector representation for each context feature and item pair.
+This approach is unable captures which specific context the different interactions occured in.
+It only captures whether or not the user or item has had an interaction in the given context.
+This differs from \textit{CSGCN-IS} since we now model context in a similar fashion to side-information, where each context value has a single vector representation, instead of having a vector representation for each context value and item pair.
```

Comment:
```suggestion
This differs from \textit{CSGCN-IS} since we now model context in a similar fashion to side-information, where each context value has a single vector representation instead of having a vector representation for each context value and item pair.
```